src/i18n/translations/en.json

```
@@ -152,6 +152,14 @@
   "lines.editor.general.publicCodeInputLabelTooltip": "Public code is what identifies the line externally towards travellers",
   "lines.editor.general.typeFormGroupTitle": "Flexible line type *",
   "lines.editor.generalTabLabel": "General",
+  "lines.editor.general.drawer": "Here is a short description of each line type.",
```

Personlig er jeg ikke noe glad i 'Here' på engelsk som oversettelse for 'Her' i slike tilfeller). Hva med eks underneath eller this?
```suggestion
  "lines.editor.general.drawer": "Underneath is a short description of each line type.",
```